neon/data-collection/gaze-mode/index.md

```diff
@@ -0,0 +1,15 @@
+# Gaze Mode
+
+In the Neon Companion App, you can select between Binocular (default) or Monocular (left or right) gaze modes. Binocular mode captures gaze data from both eyes, and is recommended for most users. Monocular mode generates gaze data from a single eye (left or right, as chosen by the user), and is advisable only for those who specifically need it.
+
+## Changing Gaze Modes
```

Comment:
This feels to short for a section and the fact that you have to re-pug the hardware is kinda important.

Suggestions: Put "After selecting a new gaze mode, be sure to unplug and re-plug the Neon device." into an info box.